hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/contracts/operations/AbstractEvmRecordingCreateOperation.java

```diff
@@ -151,7 +149,7 @@ private void spawnChildMessage(final MessageFrame frame) {
                 .sender(frame.getRecipientAddress())
                 .value(value)
                 .apparentValue(value)
-                .code(CodeFactory.createCode(inputData, Hash.EMPTY, 0, false))
+                .code(CodeFactory.createCode(inputData, 0, false))
```

Comment:
Q: why the change? Besu no longer needs the hash here I guess.
Should probably be in PR description details

Reply:
The API provided by besu changed.